.post-release-msg

```diff
@@ -1,17 +1,13 @@
-1. Upload the artifacts to the staging repository and prepare the web site:
+1. Build the web site:
 
    git checkout ${tag} && \
-   ./gradlew --no-daemon clean publish site
+   ./gradlew --no-daemon clean site
 
-2. Close and release the staging repository at:
-
-   https://oss.sonatype.org/
```

Comment:
Maybe add a step that tells the dev to wait until the artifacts are up in the Central?